pyxrf/model/fileio.py

```diff
@@ -65,11 +65,15 @@
     # registers a filestore handler for the XSPRESS3 detector
     from hxntools import handlers as hxn_handlers
 except ImportError as e:
-    logger.error('hxntools is not available: %s', e)
+    logger.error('hxntools is not available from old version: %s', e)
 
+try:
+    import vortex_handler
+except ImportError as e:
+    logger.error('handler is not loaded.')
 
 
-beamline_name = 'HXN'  # this name will be changed according to beam lines.
+beamline_name = 'SRX'  # this name will be changed according to beam lines.
```

Comment:
Good catch. beamline_name will not be used when users play with pyxrf in their laptops, this name is only used when pyxrf talks to databroker. I know this is not optimal, and my plan it to put the option at GUI level.